modules/smart-agent_nagios-status-check/conf/readme.yaml

```diff
@@ -1,47 +1,28 @@
 documentations:
```

Comment:
this doc is auto generated you have to edit its underlying yaml configuration in conf directory and run `make doc`

Reply:
I get an error when trying to generate the readme
```
terraform-signalfx-detectors:master ✓ ➭ make readmes modules/smart-agent_nagios-status-check/
./scripts/module/loop_wrapper.sh ./scripts/module/gen_doc.sh
sed: -e expression n°1, caractère 54: Fin d'intervalle invalide
make: *** [Makefile:34 : readmes] Erreur 1
```